src/main/java/ca/corefacility/bioinformatics/irida/model/Timestamped.java

```diff
@@ -11,14 +13,15 @@
 
 	/**
 	 * Get the created date of the object
-	 * 
+	 *
 	 * @return A {@link Date} object of the created date
 	 */
+	@Schema(implementation = Long.class)
```

Comment:
Could you add a description here indicating that it is Epoch time e.g.:
```Epoch time in milliseconds when the resource was created```

Reply:
Again, please see a922b821d63d07ad1000087b7880ef91f815c1a4.